pkg/manager/manager.go

```diff
@@ -73,27 +68,10 @@ type Config struct {
 
 // Manager single point of entry for the config system.
 type Manager struct {
-	Config                    Config
-	LeadershipStore           leadership.Store
-	MastershipStore           mastership.Store
-	DeviceChangesStore        device.Store
-	DeviceStateStore          state.Store
-	DeviceStore               devicestore.Store
-	DeviceCache               cache.Cache
-	NetworkChangesStore       network.Store
-	NetworkSnapshotStore      networksnap.Store
-	DeviceSnapshotStore       devicesnap.Store
-	networkChangeController   *controller.Controller
-	deviceChangeController    *controller.Controller
-	networkSnapshotController *controller.Controller
-	deviceSnapshotController  *controller.Controller
-	ModelRegistry             *modelregistry.ModelRegistry
-	TopoChannel               chan *topodevice.ListResponse
-	OperationalStateChannel   chan events.OperationalStateEvent
-	SouthboundErrorChan       chan events.DeviceResponse
-	Dispatcher                *dispatcher.Dispatcher
-	OperationalStateCache     map[topodevice.ID]devicechange.TypedValueMap
-	OperationalStateCacheLock *sync.RWMutex
+	SouthboundErrorChan     chan events.DeviceResponse
```

Comment:
How these channels used? If we disable tests, I guess we can define them locally for now as well and inject into other packages correct which need them?

Reply:
Must have been there for the tests. Nothing uses it now.